Automattic/Gravatar-SDK-Android
# Fix: Remove empty spaces in Profile component when profile description is empty

Closes #193

### Description

The Profile component should adjust its height depending on the content. The current implementation was leaving both the  `AboutMe` and the `Spacer` components when the description was empty leaving a lot of empty space in the middle of the component. 

Here's the comparison from Roborazzi:
<img width="1625" alt="Screenshot 2024-06-28 at 08 32 43" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/1210b0a0-ba1e-442e-85a8-8f18b7741b60">

Before making changes, I've added one screenshot test for LargeProfile to ensure there won't be a regression and one new test for Profile to ensure the new changes are visible. These are committed in a separate commit. 


### Testing Steps

1. Run the demo app
2. Go to the Profile tab
3. Fetch the profile with the description 
4. Confirm no changes are visible
5. Fetch the profile without a description
6. Confirm the Profile component takes less space vertically
